apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -1,66 +1,125 @@
+    {{#multiline}}
         {{#label}}
-        <span class="input-group-prepend">
-            <span class="input-group-text" id="pl-string-input-{{uuid}}-label">{{{label}}}</span>
-        </span>
+        <label
+            id="pl-string-input-{{uuid}}-label"
+            class="form-label"
+            for="pl-string-input-{{uuid}}-area"
+        >{{{label}}}</label>
         {{/label}}
-        <input
-            name="{{name}}"
-            type="text"
-            inputmode="text"
-            class="form-control pl-string-input-input"
-            size="{{size}}"
-            autocomplete="off"
-            autocorrect="off"
-            {{^editable}}disabled{{/editable}}
-            {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
-            aria-describedby="pl-symbolic-input-{{uuid}}-suffix"
-            aria-labelledby="pl-symbolic-input-{{uuid}}-label"
-            placeholder="{{placeholder}}"
-        />
-        <span class="input-group-append">
-            {{#suffix}}<span class="input-group-text" id="pl-symbolic-input-{{uuid}}-suffix">{{suffix}}</span>{{/suffix}}
-
-            {{#show_info}}
-                <a role="button" class="btn btn-light border d-flex align-items-center" data-toggle="popover" data-html="true" title="String" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
-                    <i class="fa fa-question-circle" aria-hidden="true"></i>
-                </a>
-            {{/show_info}}
+        <span id="pl-string-input-{{uuid}}" class="input-group pl-string-input">
+            <textarea
+                wrap="soft"
+                id="pl-string-input-{{uuid}}-area"
+                name="{{name}}"
+                type="text"
+                inputmode="text"
+                class="form-control pl-string-input-input multiline-input"
+                cols="{{size}}"
+                rows="2"
+                autocomplete="off"
+                autocorrect="off"
+                {{^editable}}disabled{{/editable}}
+                aria-describedby="pl-string-input-{{uuid}}-suffix"
+                placeholder="{{placeholder}}"
+            >{{raw_submitted_answer}}</textarea>
+    {{/multiline}}
+    {{^multiline}}
+        <span id="pl-string-input-{{uuid}}" class="input-group pl-string-input">
+            {{#label}}
+            <span class="input-group-prepend">
+                <span
+                    id="pl-string-input-{{uuid}}-label"
+                    class="input-group-text"
+                    for="pl-string-input-{{uuid}}-input"
+                >{{{label}}}</span>
+            </span>
+            {{/label}}
+            <input
+                id="pl-string-input-{{uuid}}-input"
+                name="{{name}}"
+                type="text"
+                inputmode="text"
+                class="form-control pl-string-input-input {{#multiline}}multiline-input{{/multiline}}"
```

Comment:
I believe this is unnecessary since this whole thing is nested under the `{{^multiline}}` case, is that right?

```suggestion
                class="form-control pl-string-input-input"
```

Reply:
Oh whoops, yes that's correct.

Reply:
Removed https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/034e1035f27db64dbf47102accab254a7e002645